packages/checkout/components/store-notices-container/store-notices.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { useRef, useEffect } from '@wordpress/element';
+import { Notice } from 'wordpress-components';
+import { sanitizeHTML } from '@woocommerce/utils';
+import { useDispatch } from '@wordpress/data';
+import { usePrevious } from '@woocommerce/base-hooks';
+import { decodeEntities } from '@wordpress/html-entities';
+import { STORE_NOTICES_STORE_KEY } from '@woocommerce/block-data';
+
+/**
+ * Internal dependencies
+ */
+import { getClassNameFromStatus } from './utils';
+import type { StoreNotice } from './types';
+
+const StoreNotices = ( {
+	context,
+	className,
+	notices,
+}: {
+	context: string;
+	className: string;
+	notices: StoreNotice[];
+} ): JSX.Element => {
+	const { removeNotice } = useDispatch( 'core/notices' );
+	const ref = useRef< HTMLDivElement >( null );
+	const { registerContainer, unregisterContainer } = useDispatch(
+		STORE_NOTICES_STORE_KEY
+	);
+
+	// Register the container with the parent.
+	useEffect( () => {
+		registerContainer( context, ref );
+		return function cleanup() {
```

Comment:
how come you've named this function? for readability or is there another reason?

Reply:
Yeah I just did it based on an example I found. I'll remove it.